README-template.md

```diff
@@ -412,6 +412,55 @@ Please note there are a few caveats:
 * Reading from views is **disabled** by default. In order to enable it, set the `viewsEnabled` configuration
   property to `true`.
 
+## Authentication
+
+The connector needs an instance of a GoogleCredentials in order to connect to the BigQuery APIs.
+
+By default on Dataproc, the connector automatically uses the cluster service account's credentials.
+
+There are multiple options to override the default behavior and to provide custom credentials:
+
+* Set the path to a service account's JSON private key in the `GOOGLE_APPLICATION_CREDENTIALS`
+  environment variable.
+* Set the path to a service account's JSON private key in the `bq.credentials.file` configuration
+  property.
+* Set a base64-encoded service account JSON private key in the `bq.credentials.key` configuration
+  property.
+* Set the fully qualified class name of a custom [access token provider]((https://github.com/GoogleCloudDataproc/spark-bigquery-connector/tree/master/bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/AccessTokenProvider.java))
+  implementation in the `bq.access.token.provider.fqcn` configuration property. The class must be
+  implemented in Java or other JVM languages such as Scala or Kotlin. It must either have a no-arg
+  constructor or a constructor accepting a single `java.util.String` parameter. This parameter can
+  be supplied using the `bq.access.token.provider.config` configuration property. If the property is
+  not set then the no-arg constructor will be called. The JAR containing the implementation class
+  should be on the cluster's classpath.
+* Define service account impersonation for specific users, specific groups, or for all
+  users by default using below properties:
+
+    - `bq.impersonation.service.account.for.user.<USER_NAME>` (not set by default)
```

Comment:
Is this Linux user or Google account user?

Reply:
added comment to clarify that it is for the linux user that runs the hive query